app/_hub/kong-inc/request-transformer-advanced/index.md

```diff
@@ -170,6 +170,14 @@ $ curl -X POST http://localhost:8001/apis \
     --data "strip_uri=false"
 ```
 
+<div class="alert alert-info.blue" role="alert">
+  **Kubernetes users:** version `v1beta1` of the Ingress specification does not
```

Comment:
@rainest Change **Kubernetes users:** to <strong>Kubernetes users:</strong> 

I believe this will fix the rendering issue. Thx!